rest-api-spec/src/main/resources/rest-api-spec/api/bulk.json

```diff
@@ -41,6 +41,18 @@
           "type": "list",
           "description" : "Default comma-separated list of fields to return in the response for updates"
         },
+        "_source": {
+          "type" : "list",
+          "description" : "True or false to return the _source field or not, or default list of fields to return"
+        },
+        "_source_exclude": {
+          "type" : "list",
+          "description" : "Default list of fields to exclude from the returned _source field"
+        },
+        "_source_include": {
+          "type" : "list",
+          "description" : "Default list of fields to extract and return from the _source field"
```

Comment:
Oh! It is the default list because it can be overridden on each sub-request, right? Maybe mention that....